scripts/component_docs_generator/helpers/docs_creator.go

```diff
@@ -0,0 +1,119 @@
+package helpers
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func CreateIntegrationDocs(csvIndices CSVIndices) {
+	inputFile, err := os.Open("filtered.csv")
+	if err != nil {
+		log.Println("Error opening input file:", err)
+		return
+	}
+	defer inputFile.Close()
+
+	csvReader := csv.NewReader(inputFile)
+	records, err := csvReader.ReadAll()
+	if err != nil {
+		log.Println("Error reading input file:", err)
+		return
+	}
+	GenerateIntegrationDocsSVG(records, csvIndices)
+	GenerateIntegrationDocs(records, csvIndices)
+}
+
+func GenerateIntegrationDocsSVG(records [][]string, csvIndices CSVIndices) {
+	for _, record := range records[1:] {
+		svgContent := record[csvIndices.SvgIndex]
+		createFiles("../../docs/assets/img/integrations", ".svg", record[csvIndices.NameIndex], svgContent)
```

Comment:
Please pass this location as an ENV var.

Reply:
@MUzairS15 yeah I would also go for passing the variable as ENV variable. I have doubt need to pass this as env variable in workflow or keep an env file and pass it from the code

Reply:
Eventually in workflow. For now in file is fine.